playbooks/common/openshift-cluster/upgrades/upgrade_nodes.yml

```diff
@@ -94,3 +94,12 @@
       name: openshift_excluder
     vars:
       r_openshift_excluder_action: enable
+
+# Tuned profile are removed when removing tuned-profiles-atomic-openshift-node rpm
+# Need to be re apply
+- name: Re-enable tuned profiles
```

Comment:
Rather than doing this in a completely separate play add it as another post_task after line 59.